pantos/client/library/entitites.py

```diff
@@ -0,0 +1,101 @@
+"""Module that defines the Pantos client library entities.
+
+"""
+import dataclasses
+import enum
+import uuid
+
+from pantos.common.blockchains.enums import Blockchain
+from pantos.common.servicenodes import ServiceNodeTransferStatus
```

Comment:
We should import `ServiceNodeTransferStatus` from `pantos.common.entities`, where it's actually defined.